File: Homework1/Homework1/Calculator.cs
namespace Homework1
{
    public static class Calculator
    {
        public static int Calculate(string operation, int val1, int val2)
        {
            var result = operation switch
            {
                "+" => val1 + val2,
                "-" => val1 - val2,
                "*" => val1 * val2,
                "/" => val1 / val2
            };
            return result;
        }
    }
}

File: Homework6/Giraffe/Giraffe/Calculator.fs
module Giraffe.Calculator

open System
open Giraffe.Operation

let inline Calculate arg1 operation arg2 =
    match operation with
    | Operation.Add -> arg1 + arg2
    | Operation.Subtract -> arg1 - arg2
    | Operation.Divide -> arg1 / arg2
    | Operation.Multiply -> arg1 * arg2